queries.sql

```diff
@@ -49,3 +49,20 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
```

Comment:
Great job @Ngufor-emmanuella 🤝

- [ ] As stated in the project requirements, kindly add all the queries into the queries.sql file